source/math/FloatBox2D.ooc

```diff
@@ -63,6 +63,18 @@ FloatBox2D: cover {
 	shrink: func ~fraction (margin: Float) -> This {
 		this pad(-margin * this height / 2.0f)
 	}
+	setSize: func (size: FloatSize2D) -> This {
```

Comment:
I don't like this name, because it makes it sound like you are only modifying the size (and knowing a box consists of a top-left position and a size, that's misleading). `resizeTo` would make things clearer.